docs/source/development/technology-stack.rst

```diff
@@ -0,0 +1,150 @@
+Technology Stack
+----------------
+
+This page lists the various technologies and libraries that the IMAP SDC
+utilizes along with a few notes on what they are used for, why they were chosen,
+and what other options were considered. It is worth noting that the development
+of the SDC began in ~2022-2023, and so many of the technology decisions reflect
+the technological ecosystem of that time.
+
+AWS
+~~~
+
+The SDC uses Amazon Web Services (AWS) as the solution for `cloud
+infrastructure <https://lasp.colorado.edu/galaxy/display/IMAP/SDC+Architecture>`_
+used for data processing. Amongst cloud vendors, AWS was chosen mostly because
+of its popularity within scientific software applications. Other cloud vendors
+such as Microsoft Azure and Google Cloud were not really considered.
+
+AWS was chosen over an on-prem architecture due to the general movement within
+the community to move data processing and storage to the cloud.
+
+The following sections describe the particular cloud services used in the SDC:
+
+CDK
+"""
+
+
+API Gateway
+"""""""""""
+
+Lambda
+""""""
+
+Batch
+"""""
+
+EventBridge
+"""""""""""
+
+RDS
+"""
+
+`RDS <https://lasp.colorado.edu/galaxy/display/IMAP/SDC+Database+Tables>`_ was
+famously chosen after an extremely lengthy discussion going back-and-forth on
+pros and cons of various database technologies (OpenSearch and DynamoDB were the
+other major considerations). RDS was ultimately chosen for being the "old
+reliable" option, as it has been extensively used and is extremely well
+documented.
+
+S3
+""
+
+
+Common Data Format
+~~~~~~~~~~~~~~~~~~
+
+The Common Data Format (CDF) was selected as the file format for IMAP data from
+requirements. CDF is a widely used data format within the Heliophsyics
+community. This decision was based purely on `requirements
+<https://lasp.colorado.edu/galaxy/display/IMAP/IMAP+SDC+to+Instrument+Team+ICD#IMAPSDCtoInstrumentTeamICD-1.3FormatStandards>`_
+and arguably would not be the top choice amongst IMAP SDC developers. As such,
```

Comment:
Could this somehow backfire on LASP for stating this publicly?

Reply:
Yeah good point. Perhaps this is overly passive aggressive lol. I can take it out

Reply:
Agree it's passive aggressive. I would add in a plug for cdflib here as well.